advanced-credential-propagation/irsa/tutorial.md

```diff
@@ -0,0 +1,432 @@
+## Pre-requisites
+
+> Check with your Domino Professional Services before using this capability. This is a different from how Domino manages
+> pod identities by default. 
+
+1. Install [Domsed](https://github.com/dominodatalab/domino-field-solutions-installations/tree/main/domsed#readme). 
+2. For users needing to assume AWS role identities create service account per user in the `domino-compute` namespace. 
+   The name of the service account should match the user-name
+
+## The IRSA process with Domino
+
+IAM stands of Identity and Access Management. It is a two step process. The first step is Authentication where
+a process has to establish its identity. The next step is Access Management where the authenticated Principal requests
+access to a resource, an IAM Role in the IRSA process.
+
+1. To enable Authentication, apply the [mutation](user-identity-based-irsa.yaml) as follows:
+
+```shell
+kubectl -f domino-platform apply -f user-identity-base-irsa.yaml
+```
+
+This yaml defines a mutation using the `domsed` which is a Domino aware mutating webhook. It does the following:
+
+a. Update the user to K8s service account mappings (see Pre-requisites)
+
+Next update the user to K8s service account mappings (see Pre-requisites)
+
+```yaml
```

Comment:
Doesnt this conflict with the direction that the SA name should match the username? And we added a thing to the mutation to use this logic too right

Reply:
I want to keep this general. We have the ability to auto-match but then I have to get into details like the naming conventions should not include a `@`. This is an explicit version . I can make a comment about the alternatives but in tutorials I think it would impact the flow of thought. What do you think

Reply:
Yea agreed lets keep it simple for the article i think, we dont have to go into every possible scenario